prober/prober.sh

```diff
@@ -0,0 +1,11 @@
+set -eEuo pipefail
+
+ID_TOKEN=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?audience=$AUDIENCE" -H "Metadata-Flavor: Google")
+
+./jvsctl token --auth-token $ID_TOKEN -e "prober" > jvs_token
+
+cat jvs_token | ./jvsctl validate --token -
+
+./jvsctl token --auth-token $ID_TOKEN -e "prober" > jvs_breakglass_token
```

Comment:
Will the script continue to run the breakglass CUJ if the previous normal CUJ fail?

Reply:
We don't need breakglass. When it's breakglass, CLI won't contact server. Remember this is _prober_, not unit test of the CLI